its/ruling/src/test/resources/expected/python-S5713.json

```diff
@@ -2,13 +2,6 @@
 'project:django-2.2.3/django/template/defaultfilters.py':[
 816,
 ],
-'project:tornado-2.3/tornado/platform/common.py':[
-75,
-84,
-],
-'project:twisted-12.1.0/twisted/internet/selectreactor.py':[
-115,
-],
 'project:twisted-12.1.0/twisted/web/http.py':[
```

Comment:
These are deprecated Error APIs. So they are somewhat FNs as now IOError does not exist anymore (since python 3.3)  instead it is now OsError and socket.error is an alias to OsError.

Reply:
I had a very quick look at this one, it seems that `IOError` and `socket.error` refer to the same symbol (`OSError` ?) this is indeed FN, but quite acceptable I feel.